tools/src/commands/PublishProdExpoHomeCommand.ts

```diff
@@ -0,0 +1,222 @@
+import { Command } from '@expo/commander';
+import JsonFile from '@expo/json-file';
+import {
+  isMultipartPartWithName,
+  parseMultipartMixedResponseAsync,
+} from '@expo/multipart-body-parser';
+import chalk from 'chalk';
+import fs from 'fs/promises';
+import fetch, { Response } from 'node-fetch';
+import nullthrows from 'nullthrows';
+import os from 'os';
+import path from 'path';
+
+import { deepCloneObject } from '../Utils';
+import { Directories, EASUpdate } from '../expotools';
+import AppConfig from '../typings/AppConfig';
+
+type ExpoCliStateObject = {
+  auth?: {
+    username?: string;
+  };
+};
+
+const EXPO_HOME_PATH = Directories.getExpoHomeJSDir();
+
+const iosPublishBundlePath = '../ios/Exponent/Supporting/kernel.ios.bundle';
+const androidPublishBundlePath = '../android/app/src/main/assets/kernel.android.bundle';
+const iosManifestPath = '../ios/Exponent/Supporting/kernel-manifest.json';
+const androidManifestPath = '../android/app/src/main/assets/kernel-manifest.json';
```

Comment:
Could you use `IOS_DIR` and `ANDROID_DIR` and also `path` to join them?
At some point I'd love to move Expo Go codebase (`ios`, `android` and `home`) into `apps/expo-go` folder, so hardcoding is not a good idea 😉